.travis.yml

```diff
@@ -1,5 +1,13 @@
+sudo: required
+dist: trusty
+
+services: docker
+
 language: java
 
 script:
- - ant;
- - ant test;
+ - docker-compose up --build -d
```

Comment:
We use docker-compose when we want to write multi-container Docker applications. You don't have to use it here and can simply build it with "docker build".